docs/runtime.md

```diff
@@ -0,0 +1,53 @@
+# Runtime
+
+Debido a que estoy trabajando con TypeScript puedo utilizar varios runtimes como podrían ser Deno, Node.js, Jitar o Nakama.
+
+## Deno
+
+Deno es un runtime basado en la v8 de Chrome, desarrollado en el lenguaje de programación Rust y que compite directamente con Node.js, tanto en funcionalidades como en casos de uso. urge para suplir una serie de defectos que en NodeJS, se habían acentuado con el paso del tiempo y el avance de las tecnologías. Algunos de estos defectos serían:
+- Adaptación a los cambios del lenguaje compleja
+- Gestión de las dependencias
+- Módulos
+
+## Node.js
+
+Node.js es un entorno de tiempo de ejecución de JavaScript. Este entorno de tiempo de ejecución en tiempo real incluye todo lo que se necesita para ejecutar un programa escrito en JavaScript. También aporta muchos beneficios y soluciona muchísimos problemas. Se ejecuta en el motor de tiempo de ejecución JavaScript V8. Utiliza un modelo de entrada y salida sin bloqueo controlado por eventos que lo hace ligero y eficiente. Puede referirse a cualquier operación, desde leer o escribir archivos de cualquier tipo hasta hacer una solicitud HTTP. 
+
+## Jitar
+
+Jitar es un runtime distribuido para JavaScript y TypeScript que automatiza la comunicación de extremo a extremo para sus aplicaciones. Puede llamar a sus funciones del servidor directamente, Jitar se encarga del resto. A diferencia de cualquier solución similar a un marco, no hay rastro de Jitar en el código de la aplicación.
```

Comment:
Conviene que enlacéis las herramientas que exponéis. De esta en concreto no había oido hablar en mi vida, pero estoy casi seguro de que no es un runtime de JS ni nada que se le parezca.